server/src/main/java/org/elasticsearch/index/snapshots/blobstore/BlobStoreIndexShardSnapshots.java

```diff
@@ -135,6 +136,8 @@ public FileInfo findNameFile(String name) {
 
     static final class ParseFields {
         static final ParseField FILES = new ParseField("files");
+        static final ParseField SEQUENCE_NUM = new ParseField("sequence_num");
```

Comment:
I think we should avoid using sequence_num. We should use either global_checkpoint, or local_checkpoint, or max_seq_no.

Reply:
Went with `global_checkpoint` now :)